apps/backpack/views.py

```diff
@@ -303,6 +303,9 @@ def pdf(request, *args, **kwargs):
     slug = kwargs["slug"]
     try:
         badgeinstance = BadgeInstance.objects.get(entity_id=slug)
+        # TODO: Check other recipient types 
```

Comment:
I'm a bit confused by this change, is this supposed to be here?

Reply:
No, i removed it